src/renderer/features/contacts/ContactFilter/model/contact-filter.ts

```diff
@@ -3,17 +3,24 @@ import { combine, createEvent, createStore, sample } from 'effector';
 import { includes } from '@shared/lib/utils';
 import { contactModel } from '@entities/contact';
 
-const componentMounted = createEvent();
+const formInitiated = createEvent();
 
 const $filterQuery = createStore<string>('');
 const queryChanged = createEvent<string>();
-const queryReset = createEvent();
 
-$filterQuery.on(queryChanged, (_, query) => query).reset(queryReset);
+sample({
+  clock: queryChanged,
+  target: $filterQuery,
+});
+
+sample({
```

Comment:
Better to swap these `samples` so you read the code like a book starting from `formInitiated` event